flake8_django/checkers/model_content_order.py

```diff
@@ -0,0 +1,107 @@
+from ast import Assign, Call, ClassDef, FunctionDef
+
+from .base_model_checker import BaseModelChecker
+from .issue import Issue
+
+
+class DJ09(Issue):
+    code = 'DJ09'
+    description = 'Order of model inner classes and standard methods does not follow Django style guide: {elem_type} should come before {before}'
+
+    def __init__(self, elem, elem_type, before):
```

Comment:
Let's use the fullname for the parameters: `element`, `element_type`, `should_be_before_element`